sections/header-topbar.liquid

```diff
@@ -26,12 +26,12 @@
 
 <style>
     .header-topbar{
-        z-index: 10;
+        z-index: auto;
     }
     .header__topbar{
         background-color: var(--color-background-color);
         color: var(--color-text-color);
-        padding: 13px 0;
+        /* padding: 13px 0; */
```

Comment:
instead of commenting remove unnecessary lines

Reply:
Ok I removed the commented lines